docs/vetomint.md

```diff
@@ -0,0 +1,199 @@
+# Vetomint
+
+## Summary
+
+Vetomint is a proof-of-stake, byzantine fault tolerant consensus algorithm
+designed to meet the special needs of Simperby. As its name suggests, Vetomint
+is a modification of Tendermint. It is designed to support
+displacing a designated consensus leader for arbitrary reasons via voting.
+Moreover, Vetomint is designed with the self-hosted nature of Simperby in mind,
+thereby runs under long timeout intervals, possibly a few days long, without
+performance penalty.
+The modifications applied to Tendermint, and the resulting properties of
+Vetomint are summarized below:
+
+### Behaviors
+
+1. Validators may cast nil prevotes before the timeout expiration to vote against
+the proposal.  
+2. If a validator collects more than 5/6 of the
+prevotes (either nil or nonnil ones), it immediately stops and casts a precommit
+based on the votes it has collected so far.
+
+### Properties
+
+1. The byzantine threshold is $<1/6$.  
+2. If all non-byzantine validators cast
+prevotes/precommits before the timeout expiration, the system is never blocked
+by timeout.  
```

Comment:
I understand your concern about the asynchronicity, but
I don't understand the suggested phrase.

How about this?

> 2. Assuming that all non-byzantine validators cast prevotes/precommits and the
messages are delivered before the timeout expiration, the system is never
blocked by timeout.

Reply:
Good, but how about re-emphasizing that we are considering timeout-by-dispute? Reader might get confused like "how does timeout ever happens if the network's fine?" This would work:
> Assuming that all non-byzantine validators cast prevotes/precommits and the messages are delivered before the timeout expiration, the system is never blocked by timeout because of possible disuptes.

Reply:
seems good to me.
I fixed it as follows:
> 2. Assuming that all non-byzantine validators cast prevotes/precommits and the
messages are delivered before the timeout expiration, the system is never
blocked by timeout by possible disuptes.